202009.0/5dc7f1b6-3cef-44e1-8302-5d4e23e0cf0c.md

```diff
@@ -133,7 +133,7 @@ $cartTransfer->addItem($cartItem1)->addItem($cartItem2);
 $items = $cartTransfer->getItems();
 ```
 
-### Checking the Required Fields
+### Checking the Required Fields (deprecated in version 3.25.0)
```

Comment:
I know that this is quite obvious thing, but this document about transfer feature and not Transfer module. Would be nice to have module name near the required version